apps/remix-ide/src/app/panels/tab-proxy.js

```diff
@@ -148,22 +151,56 @@ export class TabProxy extends Plugin {
 
     this.on('manager', 'pluginActivated', ({ name, location, displayName, icon, description }) => {
       if (location === 'mainPanel') {
-        this.addTab(
-          name,
-          displayName,
-          () => this.emit('switchApp', name),
-          () => {
-            if (name === 'home' && this.loadedTabs.length === 1 && this.loadedTabs[0].id === "home") {
-              const files = Object.keys(this.editor.sessions)
-              files.forEach(filepath => this.editor.discard(filepath))
+        const prevSession = window.localStorage.getItem("openedTabs")
+          
+        if(prevSession){
+            const tabs = JSON.parse(prevSession)
+      
+            for (const tab of tabs){
+              if(tab.name != null){
+                try{
+                  this.addTab(tab.name, tab.title, 
+                  () => {
```

Comment:
I need to find a way to switch the Tabs, this keeps erroring out. Any suggestions?